_dashboards/branding.md

```diff
@@ -90,6 +93,57 @@ mark:
 applicationTitle: "My custom application"
 ```
 
+### Condensed header
+
+The condensed header view reduces the footprint of the header and frees up space on the page by combining navigational elements into a single header bar.
+
+The current default view remains close in appearance to the two-bar header offered in the previous version of Dashboards, with minor differences. To specify the condensed header, add the configuration property `useExpandedHeader` to the `opensearch_dashboards.yml` file and set the value to `false`, as the following example illustrates.
+
+ ```yml
+# opensearchDashboards.branding:
+  # logo:
+    defaultUrl: "https://example.com/sample.svg"
+    darkModeUrl: "https://example.com/dark-mode-sample.svg"
+  # mark:
+    # defaultUrl: ""
+    # darkModeUrl: ""
+  # loadingLogo:
+    # defaultUrl: ""
+    # darkModeUrl: ""
+  # faviconUrl: ""
+  applicationTitle: "my custom application"
+  useExpandedHeader: false
+```
+
+In a future release, default behavior will become `useExpandedHeader: false`. If you want to retain the default view in subsequent releases, you can explicitly set the property to `true` in advance. You can also wait to do this when upgrading.
```

Comment:
Last sentence: Would any meaning be lost if we just said "You can also do this when upgrading"? Otherwise, I would replace "to" with "and".